app/views/transacs/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<header>
+  <%= link_to groups_path, class: 'back-btn' do %>
+    <ion-icon name="arrow-back" class="settings-btn"></ion-icon>
+  <% end %>
+  <h1 class="page-title">New Transaction</h1>
+  <%= button_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out" %>
+</header>
+
+<%= form_with model: @new_transac, method: :post do |f| %>
+<section class="fields">
+  <div class="field">
+    <%= f.text_field :name, placeholder: "Name" %>
+  </div>
+  <div class="field">
+    <%= f.number_field :amount, placeholder: "Amount in $" %>
+  </div>
+  <div class="field">
+    <%= f.collection_select :group_id, @user_groups, :id, :name %>
+  </div>
+  <%= f.hidden_field :author_id, value: current_user.id %>
+
+  
+  <%= f.submit "ADD", class: "btton" %>
+</section>
+<% end %>
```

Comment:
- [ ] It would be nice if you could improve the UI of this page so elements are well displayed on the screen as required. Alignment and spacing of elements are key to a good user experience of your app. Please reduce the font size of the heading and add a little padding on the signout button so it appears as a button instead of a badge 👍🏽
- [ ] Please also add some padding on the input fields for creating a transaction so there is some breathing space in the design. It will improve the look and feel 👍🏽

<img width="464" alt="Screenshot 2023-07-15 at 6 47 37 PM" src="https://github.com/brainskev/Budget_App/assets/50941074/399c316c-9f03-4697-8329-d1447b578cd0">